### PR TITLE
Corrected shutdown event

### PR DESCRIPTION
**Pull request by ogorzalka**
Hi Julien,

I just sent you a fix that allows the WordPress `shutdown` action to work. It's pretty much used by debug tools  (like Query Monitor) or cache extensions. Currently this hook is not active.

Have a good day !

### CODE REVIEW

File: src/Core/Application.php [hunk 1]
```diff
@@ -1280,7 +1280,8 @@ public function manage(string $kernel, $request)
         $kernel = $this->make($kernel);
 
         $response = $kernel->handle($request);
-        $response->send();
+        $response->sendHeaders();
```
Comment (jlambe) [inline src/Core/Application.php:1283]: @ogorzalka Separating the calls makes the `shutdown` hook working ? It's weird as the `send` method only calls those two methods behind the scene.

**Review by jlambe — COMMENTED** [review]
Wonder if the problem is not coming from something else ? (view code comment)

**Comment by ogorzalka** [comment]
The problem comes from this part: 

fastcgi_finish_request();

It break the response before the shutdown hook

Well, I think it could be possible to call the shutdown action before this function no ?

**Comment by ogorzalka** [comment]
Hi @jlambe ,

Have you taken a look at this?

Do we really need to finish the request ?

If you want to test, install Query Monitor and you can see that on the front office it is not working because of this.

Thanks, have a good day :)

**Comment by jlambe** [comment]
Hi @ogorzalka,

this is all working fine now 👍 See screenshot below:

themosis_query_monitor

**Pull request closed without merging**